src/test/resources/Features/verifyErrorsAndWarningsOnListingUpload.feature

```diff
@@ -59,17 +59,31 @@ Feature: Verify expected errors and warnings display on inspect screen after lit
+    |An unrecognized character was found in Test Participant "ID06" "Professional Experience Months" "88�". The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Professional Experience Months|
+    |An unrecognized character was found in Test Task "A1.3" "Task Errors Standard Deviation" "9.75�". The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Errors Standard Deviation|
+    |An unrecognized character was found in Test Task "A1.3" "Task Rating Standard Deviation" "1.00�". The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Rating Standard Deviation|
+    |An unrecognized character was found in Test Task "A1.3" "Task Errors" "9.75�". The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Errors value|
+    
+    Scenario Outline: Verify warnings for bad input values in Test task and Participant fields on upload-inspect for 2015 listings
+    Given I'm logged in as "ROLE_ACB"
+    And I am on Upload Certified Products page
+    When I upload a 2015 listing with bad input in "<Field>"
+    Then I see upload successful message
+    When I go to Confirm Pending Products Page
+    And I inspect listing details for listing with bad inputs
+    Then I see warning "<warning>" for non-integer numeric number in "<field>"
     Examples:
     |warning|field|
-    |An unrecognized character was found in Test Task A1.3 Task Errors Standard Deviation value. The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Errors Standard Deviation|
-    |An unrecognized character was found in Test Task A1.3 Task Rating Standard Deviation value. The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Rating Standard Deviation|
-    |An unrecognized character was found in Test Task A1.3 Task Errors value. The value must be only a numeric value. You can correct it within the field itself on the Edit Certified Product screen or modify it in the csv file and upload again.|Task Errors value|
-       
\ No newline at end of file
+    |A non-integer numeric number was found in Test Participant "ID01" "Computer Experience Months" "250.75". The number has been rounded to "251".|Computer Experience Months|
+    |A non-integer numeric number was found in Test Task "A1.1" "Task Path Deviation Observed" "9.75". The number has been rounded to "10".|Task Path Deviation Observed|
```

Comment:
Would recommend having one of these round down, so we make sure we're actually rounding correctly, instead of just "Math.ceiling" them